### PR TITLE
LRDOCS-2562 Create a decription to repository-entry-browser tag

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

**Comment by robertoDiaz** [comment]
hey @jhinkey 

I've try to follow the instructions from https://dev.liferay.com/participate/taglib-doc-guidelines. 

If I did something wrong, please tell me. 

Thanks!!!

**Comment by jhinkey** [comment]
Started reviewing. Thanks.

File: modules/apps/collaboration/item-selector/item-selector-taglib/src/main/resources/META-INF/liferay-item-selector.tld [hunk 1]
```diff
@@ -11,15 +11,18 @@
 	<short-name>liferay-item-selector</short-name>
 	<uri>http://liferay.com/tld/item-selector</uri>
 	<tag>
+		<description>A User interface to select groups<![CDATA[<br /><br />Example:<br /><img src="../images/item-selector/group-selector.png"/>]]></description>
```
Comment (jhinkey) [inline modules/apps/collaboration/item-selector/item-selector-taglib/src/main/resources/META-INF/liferay-item-selector.tld]: By groups, do you mean scope groups, similar to an [application scope](https://dev.liferay.com/discover/portal/-/knowledge_base/7-0/application-scope)?

**Comment by robertoDiaz** [comment]
Yes.

This way you could select and use images from a different  site, for example.

**Comment by jhinkey** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/39385

**Pull request closed without merging**